### PR TITLE
Fix: Remove locale from PayPal SDK query parameters

### LINKED ISSUE #6790: PayPal should manage buyers locale parameter
## Details
PayPal Donations: get_locale can provide an incorrect locale to PayPal.
If the get_locale function returns anything other than a locale supported by Paypal, PayPal Donations throws a 400 error.

## Expected Behavior
-Paypal handles setting user locale via browser

## Steps to Reproduce
- Set your site's language to Greek (WP uses locale "el")
- Visit a donation form page and select Paypal Donations as the payment gateway.
- Notice that the console throws a 400 error for the Paypal script. The locale is set to "el" but Paypal requires "el_GR".

## Acceptance Criteria

-   [x] Paypal defines the local parameter based on users browser.
-   [x] No errors occur from mismatched locales.

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves #6790

## Description

Errors occurred when locale information being sent through Paypals SDK query parameters do not match the expected Paypal format. Per Paypals recommendation this PR removes the local parameter from being sent so that Paypal can automatically set this information.

Reference: https://developer.paypal.com/sdk/js/configuration/#link-queryparameters
Screen Shot 2023-04-28 at 2 41 48 PM

## Affects

Paypal Donations

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204525071226798

### CODE REVIEW

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php [hunk 1]
```diff
@@ -142,12 +142,12 @@ public function loadPublicAssets()
 
         /**
          * List of PayPal query parameters: https://developer.paypal.com/docs/checkout/reference/customize-sdk/#query-parameters
+         * @unreleased Removes locale from parameters.
```
Comment (kjohnson) [inline src/PaymentGateways/PayPalCommerce/ScriptLoader.php:145]: Can you make this past tense, so it reads as a change log?

**Review by kjohnson — APPROVED** [review]
Simple enough. I guess that makes sense that we don't need to be controlling that value, since it defaults to the browser's local.

File: src/PaymentGateways/PayPalCommerce/ScriptLoader.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply src/PaymentGateways/PayPalCommerce/ScriptLoader.php]: Resolved: https://github.com/impress-org/givewp/pull/6791/commits/0135ebd2b23fb5e18277467cc21c9e789bb22ca7

**Review by rickalday — APPROVED** [review]
Passed manual and automated QA tests

**Pull request merged**